css/styles.css

```diff
@@ -67,8 +67,9 @@ ul li {
 
 #about-me > ul > li > ul > li > ul > li {
   background-color: white;
-  padding: 8px;
```

Comment:
Great job on your project! I just wanted to provide some feedback based on the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md). Here are a few things to consider:

- [ ] It would be great if we could implement a mobile-first approach for this project. I noticed that you're currently using `max-width: 768px` to apply styles for mobile devices, but we need to make sure that the desktop styles are included within a media query using `min-width:768px`.

   ![image](https://github.com/flov3rh/portfolio/assets/107952724/78a56ad9-89b3-4881-b811-03f6488d66e6)
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/2ab4ee3e-7ecf-4c79-802b-3639e2685b44)